internal/controller/postgrescluster/pgbackrest.go

```diff
@@ -1512,12 +1520,13 @@ func (r *Reconciler) reconcilePGBackRest(ctx context.Context,
 	}
 
 	// reconcile the RBAC required to run pgBackRest Jobs (e.g. for backups)
-	sa, err := r.reconcilePGBackRestRBAC(ctx, postgresCluster)
-	if err != nil {
-		log.Error(err, "unable to create replica creation backup")
-		result.Requeue = true
-		return result, nil
-	}
+	// K8SPG-698: it should happen earlier
```

Comment:
can we removed comments

Reply:
https://github.com/percona/percona-postgresql-operator/pull/1125/commits/f1f4ce304e4a2d82436d5acc12d6022ae0db7fc8